lib/index.ts

```diff
@@ -148,43 +148,38 @@ export const imagePath = (app: string, file: string) => {
  * @return {string} URL with webroot for a file in an app
  */
 export const generateFilePath = (app: string, type: string, file: string) => {
-	const isCore = window?.OC?.coreApps?.indexOf(app) !== -1
+	const isCore = window?.OC?.coreApps?.includes(app)
+	const isPHP = file.slice(-3) === 'php'
```

Comment:
This is also equivalent to `file.substring(file.length - 3) === 'php'`, good catch to reuse it